chatbot-core/data/chunking/chunking_utils/logger.py

```diff
@@ -0,0 +1,15 @@
+"""Logger configuration for chunking scripts."""
```

Comment:
Please follow the same pattern
https://github.com/jenkinsci/resources-ai-chatbot-plugin/pull/8/files#diff-b5b57beabc3cd0c1c74ea5fde8a1bf77980e81008804669088640b21150b2947R8

If this file is not being used, please remove it.